app/file.go

```diff
@@ -856,6 +855,11 @@ func (t *UploadFileTask) postprocessImage() {
 		}
 		writeJPEG(preview, t.fileinfo.PreviewPath)
 	}()
+
+	go func() {
+		defer wg.Done()
+		t.fileinfo.MiniPreview = model.GenerateMiniPreviewImage(decoded)
```

Comment:
@hmhealey can you elaborate on what Go convention does this violate? Not sure what could be racy here, since indeed we are blocking until struct is updated before existing the routine.

Reply:
I think modifying `t.fileinfo` here might be picked up as racey by some of the Go tooling since this goroutine is modifying it while the other goroutines fired off above may be looking at it. I'm not sure if there's anything wrong with the actual logic here, but the tooling might complain about that at some point.